src/service/private.overseas.entity.details.ts

```diff
@@ -30,3 +30,32 @@ export const getPrivateOeDetails = async (
 
   return response.resource;
 };
+
+export const getBeneficialOwnerPrivateData = async (
+  req: Request,
+  transactionId?: string,
+  overseasEntityId?: string
+): Promise<BeneficialOwnersPrivateDataResource | undefined> => {
+  const response = await makeApiCallWithRetry(
+    "overseasEntity",
+    "getBeneficialOwnerPrivateData",
+    req,
+    req.session as Session,
+    transactionId,
+    overseasEntityId
+  );
+
+  if (response.httpStatusCode !== 200 && response.httpStatusCode !== 404) {
```

Comment:
lines 48 to 58 repeat what is in get oe details function. Not sure it's worth having a common method to cut down on code duplication?